contracts/staking/test/unit_tests/mixin_stake_storage_test.ts

```diff
@@ -0,0 +1,154 @@
+import { blockchainTests, expect, Numberish } from '@0x/contracts-test-utils';
+import { StakingRevertErrors } from '@0x/order-utils';
+import { BigNumber } from '@0x/utils';
+
+import { StoredBalance } from '../utils/types';
+
+import { artifacts, TestMixinStakeStorageContract } from '../../src';
+
+blockchainTests.resets('MixinStakeStorage unit tests', env => {
+    let testContract: TestMixinStakeStorageContract;
+    let defaultSyncedBalance: StoredBalance;
+    let defaultUnsyncedBalance: StoredBalance;
+
+    before(async () => {
+        testContract = await TestMixinStakeStorageContract.deployFrom0xArtifactAsync(
+            artifacts.TestMixinStakeStorage,
+            env.provider,
+            env.txDefaults,
+            artifacts,
+        );
+        await testContract.setCurrentEpoch.awaitTransactionSuccessAsync(CURRENT_EPOCH);
+        defaultSyncedBalance = {
+            isInitialized: true,
```

Comment:
What if we added one test for each function where we pass in these values but with `isInitialized: false`, then we assert that they turn to `true` after the operation?

Reply:
As far as I can tell, the only case where a balance passed into `_moveStake` or `_increment/decrement...` would have `isInitialized: false` is if the other fields are zero, so I'll add tests for this